jonathanmusila/Quesioner-React
# #user landing page

#### What does this PR do?
- create a user landing page with the current meetups

#### Description of Task to be completed?
- [x] install axios for fetching routes

#### How should this be manually tested?
- ensure that you have installed node and node package manager(npm) in your machine
- git clone https://github.com/jonathanmusila/Quesioner-React.git
- run the command npm install to install dependancies
- run command npm start to fire up the project
- got to the page `/index`
#### Any background context you want to provide?
None
